cmd/root.go

```diff
@@ -149,3 +152,23 @@ func versionCmd() *cobra.Command {
 		},
 	}
 }
+
+// outputJson print the output in json format
+func outputJson(data interface{}) {
+	jsonData, err := json.Marshal(data)
+	if err != nil {
+		log.Default().Println("Error converting to JSON", err)
+	}
+	s.Stop()
+	fmt.Println(string(jsonData))
```

Comment:
Move `s.Stop() `to Defer and add return on `err != nil` otherwise `fmt.Println` will return nil pointer